www/docs/client/cors.md

```diff
@@ -10,15 +10,19 @@ If your API reside on a different origin than your front-end and wish to send co
 The arguments provided to the fetch function used by tRPC can be modified as follow.
 
 ```ts title='app.ts'
-import { createTRPCClient } from '@trpc/client';
+import { createTRPCProxyClient, httpBatchLink } from '@trpc/client';
```

Comment:
Was not sure if `httpBatchLink` is something the docs should default to

Reply:
Also, in this specific case, I noticed that `createTRPCClient` was deprecated for `createTRPCProxyClient` so I made that change